### DISCUSSION TITLE
Glow Light Tracking

### DISCUSSION THREAD

**Discussion by ronaldolds7** [matteobruni/tsparticles#3198] [Show and tell]
I was wondering if you have ever created something like what can be found as a background animation on the site:  https://lusion.co?  I would like to get a copy or instructions on how to create this effect.  I'm particularly interested in how the curser (pointer) attracts a light in the background that follows the cursor and lights up the moving objects in the background.  Can you help me with this request?

**Comment by matteobruni** [comment]
This is the closest effect you can have with a fake light source. This is not WebGL so the light is something hard to implement: https://codepen.io/matteobruni/pen/YzqXwRr

You need to load an additional plugin: `tsparticles-interaction-light`
